modules/Administration/language/en_us.lang.php

```diff
@@ -785,21 +785,28 @@
     'LBL_TOTAL_FILES' => 'Total files',
     'LBL_REMOVED_FILES' => 'Removed files',
 
-    'LBL_JJWG_MAPS_ADMIN_HEADER' => 'Google Maps',
-    'LBL_JJWG_MAPS_ADMIN_DESC' => 'Manage your geocoding, testing geocoding, view geocoding result totals and configure advanced settings.',
-    'LBL_JJWG_MAPS_ADMIN_CONFIG_TITLE' => 'Google Maps Settings',
-    'LBL_JJWG_MAPS_ADMIN_CONFIG_DESC' => 'Configuration settings to adjust your Google Maps',
-    'LBL_JJWG_MAPS_ADMIN_GEOCODED_COUNTS_TITLE' => 'Geocoded Counts',
-    'LBL_JJWG_MAPS_ADMIN_GEOCODED_COUNTS_DESC' => 'Shows the number of module objects geocoded, grouped by geocoding response',
-    'LBL_JJWG_MAPS_ADMIN_DONATE_TITLE' => 'Donate to this Project',
-    'LBL_JJWG_MAPS_ADMIN_DONATE_DESC' => 'Please consider donating to this project!',
+    'LBL_GOOGLE_SUITE_ADMIN_HEADER'               => 'Google Suite',
```

Comment:
Please remove spaces before `' => '`

Reply:
Hi @horus68 I have added a pull request to resolve this, please check to make sure this is working.